test/benchee_test.exs

```diff
@@ -225,6 +226,73 @@ defmodule BencheeTest do
     refute output =~ ~r/compar/i
   end
 
+  test "integration profiling can be deactivated" do
+    output =
+      capture_io(fn ->
+        Benchee.run(
+          %{"Sleeps" => fn -> :timer.sleep(10) end},
+          Keyword.merge(
+            @test_configuration,
+            profile_after: false
+          )
+        )
+      end)
+
+    refute output =~ ~r/Profiling.+with/
+    refute output =~ ~r/Profile done/
+  end
+
+  @default_profiler :cprof
```

Comment:
Since this module attribute is duplicated here: https://github.com/bencheeorg/benchee/pull/317/files#diff-91ae4279786c23b5c7db654372915bfeR29, what do you think about exposing that as a function and then calling that function here to get the default profiler instead of duplicating it?

Reply:
Hey, nice catch Devon! Thanks for pointing it out, I'll change it as you suggested. I agree it's better to not have a duplicated value which could end up out of sync if someone forgets to update both :)